tom_desc/elasticc2/views.py

```diff
@@ -9,7 +9,6 @@
 
 import numpy
 import astropy.time
-import light_curve
```

Comment:
This needs to stay, as it's used below.  (See class LtcvFeatures.)

Did it give you an import error?  If so, it means you need to rebuild your docker images.

Reply:
I fixed that